tutorials/quickstart/tutorial.md

```diff
@@ -183,7 +184,7 @@ System.Exception: BOOM
    at <long stack trace>
 ```
 
-Ten seconds later, text will flash past again, warning of a 20-second delay. Twenty seconds later, the text will flash again, warning of a 30-second delay. And finally, 30 seconds after that, text will flash by again, ending in red ERROR message:
+Ten seconds later, text will flash past again, warning of a 20-second delay. Twenty seconds later, the text will flash again, warning of a 30-second delay. And finally, 30 seconds after that, text will flash by again, ending with a red ERROR message:
```

Comment:
We need to modify the whole file for the 2-second delay, especially since the delays are in the log messages as well. It's outside the diff so I can't do a suggestion.

Reply:
I changed the text to show the 2 second intervals instead of 10.